variables.tf

```diff
@@ -37,13 +37,17 @@ variable "endpoint_type" {
 
 variable "rotation_interval_month" {
   type        = number
-  description = "Interval in months to rotate the Key"
+  description = "The key rotation time interval in months. Rotation policy cannot be set for standard key, so value is ignored if var.standard_key is true"
   default     = 1
+  validation {
+    condition     = var.rotation_interval_month <= 12 && var.rotation_interval_month >= 0
```

Comment:
Should be `var.rotation_interval_month >= 1` for the lower bound.

Reply:
good catch - fixed